tests/hardhat/Fork/supply.ts

```diff
@@ -0,0 +1,325 @@
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+
+      // setup to liquidate the second account(acc2) with first account(acc1)
+      await comptroller.setMinLiquidatableCollateral(0);
+      await expect(vUSDT.connect(acc2Signer).borrow(usdtBorrowAmount)).to.be.emit(vUSDT, "Borrow");
+      await priceOracle.setDirectPrice(usdd.address, convertToUnit("1", 13));
+
+      const [err, liquidity, shortfall] = await comptroller.callStatic.getAccountLiquidity(acc2);
+      expect(err).equals(0);
+      expect(liquidity).equals(0);
+      expect(shortfall).greaterThan(0);
+
+      const borrowBalance = await vUSDT.borrowBalanceStored(acc2);
+      const closeFactor = await comptroller.closeFactorMantissa();
+      const maxClose = (Number(borrowBalance) * Number(closeFactor)) / 2e18;
+      let result = vUSDT.connect(acc1Signer).liquidateBorrow(acc2, maxClose, vUSDD.address);
+      await expect(result).to.emit(vUSDT, "LiquidateBorrow");
+
+      // Mine 300,000 blocks
+      await mine(blocksToMint);
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+
+      // Setup for healAccount(acc2)
+      await priceOracle.setDirectPrice(usdd.address, convertToUnit(1, 10));
+      await priceOracle.setDirectPrice(usdt.address, convertToUnit(1, 18));
+
+      const [err2, liquidity2, shortfall2] = await comptroller.callStatic.getAccountLiquidity(acc2);
+      expect(err2).equals(0);
+      expect(liquidity2).equals(0);
+      expect(shortfall2).greaterThan(0);
+      await comptroller.setMinLiquidatableCollateral(convertToUnit(1, 22));
+
+      result = comptroller.connect(acc1Signer).healAccount(acc2);
+      await expect(result).to.emit(vUSDT, "RepayBorrow");
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+
+      const totalBal = await vUSDT.balanceOf(acc1);
+      await expect(vUSDT.connect(acc1Signer).redeem(totalBal)).to.emit(vUSDT, "Redeem");
+
+      // Mine 300,000 blocks
+      await mine(blocksToMint);
+
+      // Accural all the interest till latest block
+      await vUSDT.accrueInterest();
+
+      // Assert current exchange rate
+      exchangeRate = await vUSDT.callStatic.exchangeRateCurrent();
+      calculatedRate = await calculateExchangeRate();
+      expect(exchangeRate).closeTo(calculatedRate, 1);
+    });
+
+    it("Two users Mint, one redeems", async () => {
```

Comment:
I would rename to "Three users Mint, one redeems"

Reply:
Fixed, 4651fb2039bb0f5ad7bc667b2269087b10f7b6f0